agent/custom/bbc_start.py

```
@@ -80,15 +80,18 @@ def run(self, context: Context, argv: CustomAction.RunArg) -> CustomAction.RunRe
             
             # 步骤1: 检查BBC进程是否存在
             mfaalog.info("[StartBbc] 步骤1: 检查BBC状态...")
-            bbc_proc = bbc_manager._find_bbc_process()
+            
+            # 获取或创建 Manager 实例（进程级单例）
```

Ruff (0.15.10)

[warning] Comment contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF003)